### PR TITLE
UG - Add expected output for pair, unpair and check property

### PR DESCRIPTION
**Pull request by ngdeqi**
+ delete common error messages table

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -130,6 +135,7 @@ Lists only selected details of all the properties, depending on the TAG. The com
 * `list -property t/` This is for unit type
 * `list -property -short` This is for the shorthand version(displays address, price and unit type)
 
+---
```
Comment (wilsonngja) [inline docs/UserGuide.md:138]: Good effort in trying to make a clear distinction between different parts. 👍🏼

File: docs/UserGuide.md [hunk 2]
```diff
@@ -209,9 +245,17 @@ Unpairs the client from the specified property, to record that the client is no
 
 <u>Example</u>: `unpair ip/1 ic/5`
 
+<u>Expected Output</u>:
+```
+Unpairing the following client and property: 
+	Kujou Jotaro and 25 Lower Kent Ridge Rd, S119081
+```
+---
+
```
Comment (wilsonngja) [inline docs/UserGuide.md:254]: Good job in following up with the post PE-D feedback.

**Review by wilsonngja — COMMENTED** [review]
Good work with the great response to the feedback given and constantly trying to make improvement. The addition is well organised and tellies with the output in our code. LGTM :D

File: docs/UserGuide.md [hunk 3]
```diff
@@ -173,8 +194,12 @@ Deletes the specified client from the client list and subsequently deletes any p
 
 <u>Example:</u> `delete -client ic/3`
 
+---
```
Comment (OVReader) [inline docs/UserGuide.md:197]: Good use of divider to make UG neater and easier to read.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -200,6 +229,13 @@ Pairs the client to the specified property, to record that the client is renting
 
 <u>Example</u>: `pair ip/1 ic/5`
 
+<u>Expected Output</u>:
+```
+Pairing the following client and property:
```
Comment (OVReader) [inline docs/UserGuide.md:234]: Thank you for updating expected outputs to fit the latest implementations

**Review by OVReader — APPROVED** [review]
Overall nicely done! LGTM

**Pull request merged**